app/domain/authentication/authn_jwt/identity_providers/add_prefix_to_identity.rb

```diff
@@ -0,0 +1,92 @@
+require 'command_class'
+
+module Authentication
+  module AuthnJwt
+    module IdentityProviders
+      # Add policy prefix to identity, for examples:
+      # 1. Input: identity_prefix=host identity=host_test1
```

Comment:
I think that if we have UT coverage the examples are redundant here

Reply:
I think it is more easy to understand the logic for the comment